src/Widgets/NotificationEntry.vala

```diff
@@ -69,7 +70,8 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
 
 
         var time_label = new Gtk.Label (Granite.DateTime.get_relative_datetime (notification.timestamp)) {
-            margin_end = 6
+            margin_end = 6,
+            halign = Gtk.Align.END
```

Comment:
I'm not certain this does anything since the label is pushed to the end by an hexpand already